go.mod

```diff
@@ -1,27 +1,89 @@
 module github.com/3scale/apicast-operator
 
-go 1.13
+go 1.17
 
 require (
 	github.com/RHsyseng/operator-utils v0.0.0-20200213165520-1a022eb07a43
```

Comment:
Think we can bump this as well to 	github.com/RHsyseng/operator-utils v1.4.9 to stay in line with what's coming to 3scale operator?

Reply:
done